components/authz-service/engine/opa/opa.go

```diff
@@ -425,12 +429,24 @@ func (s *State) RulesForProject(
 }
 
 // ListProjectMappings returns a map of all the rules for each projectID.
-func (s *State) ListProjectMappings(ctx context.Context) (map[string][]engine.Rule, error) {
-	rs, err := s.evalQuery(ctx, s.queries[listProjectMapQuery], map[string]interface{}{}, s.v2Store)
-	if err != nil {
-		return nil, &ErrEvaluation{e: err}
+func (s *State) ListProjectMappings(ctx context.Context) (map[string][]v2.Rule, error) {
+	items := s.ruleStore.Items()
+	s.log.Infof("HEY! 2. got this %#v in items", items)
+	// nothing stored while on v1 or v2.0
+	if len(items) == 0 {
+		return map[string][]v2.Rule{}, nil
+	}
+	projectRules := make(map[string][]v2.Rule, len(items))
+	for project, item := range items {
+		rules, ok := item.Object.([]v2.Rule)
```

Comment:
you can `fmt.Printf("object type is %T\n", item)` to check the type -- it might not be `[]v2.Rule`?

Reply:
ya i've tested with logging and it's coming back as the correct type.